Mathlib/AlgebraicGeometry/Scheme.lean

```diff
@@ -177,27 +189,36 @@ lemma preimage_comp {X Y Z : Scheme.{u}} (f : X ⟶ Y) (g : Y ⟶ Z) (U) :
 
 /-- The forgetful functor from `Scheme` to `LocallyRingedSpace`. -/
 @[simps!]
-def forgetToLocallyRingedSpace : Scheme ⥤ LocallyRingedSpace :=
-  inducedFunctor _
--- deriving Full, Faithful -- Porting note: no delta derive handler, see https://github.com/leanprover-community/mathlib4/issues/5020
+def forgetToLocallyRingedSpace : Scheme ⥤ LocallyRingedSpace where
+  obj := toLocallyRingedSpace
+  map := Hom.toLRSHom
 
 /-- The forget functor `Scheme ⥤ LocallyRingedSpace` is fully faithful. -/
-@[simps!]
+@[simps! preimage_toLRSHom]
```

Comment:
```suggestion
@[simps preimage_toLRSHom]
```
Aside: the generated lemma uses `toHom_1` as variable name in spite of the simps projection. Is this expected?

Reply:
I dunno. But I think it wouldn't cause much trouble here?